homeassistant/components/tts/__init__.py

```diff
@@ -92,6 +94,16 @@ def _deprecated_platform(value):
     return value
 
 
+def valid_base_url(value: str) -> str:
+    """Validate base url, return value."""
+    url = yarl.URL(cv.url(value))
+
+    if url.path != "/":
```

Comment:
Shouldn't this explicitly validate a non empty protocol? Or does normalize_url do that?

Reply:
that's already taken care off by `cv.url`.